integration-tests/mssql_test.go

```diff
@@ -53,6 +53,7 @@ var _ = Describe("MSSQL", Label("MSSQL"), func() {
 			"rds_subnet_group":           "some-rds-subnet-group",
 			"instance_class":             "some-instance-class",
 			"max_allocated_storage":      999,
+			"publicly_accessible":        false,
```

Comment:
nit-pick: this property is not needed. The default value is propagate

Reply:
Feel free to remove it. I have the tendency to add every new field to some of the maps in the tests (required, optional, defaults, etc.)